README.md

```diff
@@ -150,6 +150,10 @@ Content-Type: application/json
         "amount_available": 14500
         "amount_submitted": 0
     }
+    "settlement_summary": {
+        "captured_date": "2016-01-15",
```

Comment:
Unfortunately, yes - Worldpay does not return the time of capture, just the date. It's returned to us as `bookingDate` in the payload of the "payment captured" notification. It's true that we could've expressed this date as `2016-01-15T00:00:00Z`, for example, but I guess rather than returning potentially misleading information, it's better to stick to what we know for sure.